overlord/snapstate/snapstate.go

```diff
@@ -302,8 +306,68 @@ func Remove(s *state.State, name string) (*state.TaskSet, error) {
 
 // Rollback returns a set of tasks for rolling back a snap.
 // Note that the state must be locked by the caller.
-func Rollback(s *state.State, snap, ver string) (*state.TaskSet, error) {
-	return nil, fmt.Errorf("rollback not implemented")
+func Rollback(s *state.State, name, ver string) (*state.TaskSet, error) {
```

Comment:
@pedronis Yeah, that is a really good question. So far we only keep a single previous revision so you get only one rollback. Once we support more then of course  `RollbackR` in the snapst is no longer good and it needs to become a list or a map. I can do this now so that we are ready?